backend/modules/cohorts/repository.py

```
@@ -485,12 +485,12 @@ def get_student_cohort_simulations(
         if not enrollment:
             return []
         
-        return self.db.query(CohortSimulation, Scenario).join(
-            Scenario, CohortSimulation.simulation_id == Scenario.id
+        return self.db.query(CohortSimulation, Simulation).join(
+            Simulation, CohortSimulation.simulation_id == Simulation.id
         ).filter(
             CohortSimulation.cohort_id == cohort_id,
-            Scenario.is_draft == False,
-            Scenario.status == "active"
+            Simulation.is_draft == False,
```

Ruff (0.14.8)

Avoid equality comparisons to `False`; use `not Simulation.is_draft:` for false checks

Replace with `not Simulation.is_draft`

(E712)